packages/app/src/providers/h5grove/h5grove-api.ts

```diff
@@ -50,50 +49,81 @@ export class H5GroveApi extends DataProviderApi {
     this.fetcher = createAxiosFetcher(
       axios.create({
         adapter: 'fetch',
-        baseURL,
```

Comment:
All `fetch` calls now concatenate `this.baseUrl` to form absolute URLs, so the axios instance no longer needs to hold on to it.